packages/strapi-plugin-graphql/services/build-aggregation.js

```diff
@@ -194,8 +194,8 @@ const createAggregationFieldsResolver = function(model, fields, operation, typeC
       if (model.orm === 'bookshelf') {
         return model
           .query(qb => {
-            // apply filters without pagination limit
-            buildQuery({ model, filters: _.omit(filters, ['limit']) })(qb);
+            // apply filters
+            buildQuery({ model, filters })(qb);
```

Comment:
why don't you use the convertion here too ?

Reply:
It's already converted by [exists code](https://github.com/strapi/strapi/blob/f7a8d06f54f206ac8cbb6bd2b6f31d74a129582a/packages/strapi-plugin-graphql/services/build-aggregation.js#L179).

Reply:
OK gotcha. I think you can refactor the code in the second part to look exactly the same :) calling convertRest before the if mongose and if booskehf. Hope this makes sense :)